sources/gov/kaunas/educational_institutions.yml

```diff
@@ -0,0 +1,35 @@
+id: "gov/kaunas/municipality/educational_institutions"
+title: "Educational institutions"
+description: "CSV file containing data about Kaunas educational institutions"
+type: "source"
+source:
+  - "https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
+  - type: "csv"
+    sep: ","
```

Comment:
This should no be a list, just dict or string. If you pass string wtih `.csv`, type will be inferred from it. Also you can use scheme like this: `csv+https://...` to specify type in string form.